monitor_node:
  ros__parameters:
    # SSH credentials
    username: "root"
    password: "dsl!auv"
    filepath: "/tmp/linkstate_current.json"
    passive_timer: 5.0


    # MINI MAC -> PAYLOAD IP
    #   Payloads

    #  Base Stations
    mac_to_ip_map: >
      {
        "00:30:1a:3b:14:c9": "10.19.30.100",
        "00:30:1a:3b:15:31": "10.19.30.101",
        "00:30:1a:3b:15:2d": "10.19.30.102",
        "00:30:1a:3b:14:d1": "10.19.30.103",
        "00:30:1a:3b:14:ad": "10.19.30.104",
        "00:30:1a:3b:14:9d": "10.19.30.2",
        "00:30:1a:3b:14:cd": "10.19.30.3"
      }
